vpr/src/base/netlist_writer.cpp

```diff
@@ -2572,3 +2587,48 @@ std::string escape_sdf_identifier(const std::string identifier) {
 std::string join_identifier(std::string lhs, std::string rhs) {
     return lhs + '_' + rhs;
 }
+
+} // namespace
+
+//
+// Externally Accessible Functions
+//
+
+///@brief Main routine for this file. See netlist_writer.h for details.
+void netlist_writer(const std::string basename, std::shared_ptr<const AnalysisDelayCalculator> delay_calc, struct t_analysis_opts opts) {
+    std::string verilog_filename = basename + "_post_synthesis.v";
+    std::string blif_filename = basename + "_post_synthesis.blif";
+    std::string sdf_filename = basename + "_post_synthesis.sdf";
+
+    VTR_LOG("Writing Implementation Netlist: %s\n", verilog_filename.c_str());
```

Comment:
Both the netlist_writer and merged_netlist_writer say the same thing: "Writing Implementation Netlist".  It would be better to explain what kind of netlist, as there should be some difference between them to justify two different routines.